lib/driver.js

```diff
@@ -156,12 +155,12 @@ class XCUITestDriver extends BaseDriver {
   }
 
   async onSettingsUpdate (key, value) {
-    if (key !== 'nativeWebTap') {
+    if (key !== 'nativeWebTap' && key !== 'nativeWebTapStrict') {
       return await this.proxyCommand('/appium/settings', 'POST', {
         settings: {[key]: value}
       });
     }
-    this.opts.nativeWebTap = !!value;
+    this.opts[key] = !!value;
```

Comment:
I'm not quite sure this is not going to break anything...

Reply:
The only two keys that get through to here are the native web tap ones. This is not a problem.